application/src/main/java/org/opentripplanner/service/osminfo/OsmInfoGraphBuildRepository.java

```diff
@@ -31,4 +38,6 @@ public interface OsmInfoGraphBuildRepository extends Serializable {
    * Find the platform which relates to an area
    */
   Optional<Platform> findPlatform(Area area);
+
+  Collection<TurnRestriction> getTurnRestrictions();
```

Comment:
```suggestion
  Collection<TurnRestriction> listTurnRestrictions();
```

Sorry, I found one more thing. According to our [naming conventions](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/doc/dev/decisionrecords/NamingConventions.md) it's `list**`.